src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php

```diff
@@ -120,45 +131,41 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'disabled' => empty($productId),
                 ],
             ])
-            ->add('preview', IconButtonType::class, [
-                'label' => $this->trans('Preview', 'Admin.Actions'),
-                'icon' => 'remove_red_eye',
-                'attr' => [
-                    'class' => 'btn-outline-secondary preview-url-button',
-                    'data-seo-url' => $seoUrl,
-                    'disabled' => empty($productId),
-                ],
-            ])
             ->add('new_product', IconButtonType::class, [
-                'label' => $this->trans('New product on experimental page', 'Admin.Catalog.Feature'),
+                'label' => $this->trans('Add new product', 'Admin.Catalog.Feature'),
                 'type' => 'link',
                 'icon' => 'add_circle_outline',
                 'attr' => [
-                    'class' => 'btn-outline-secondary new-product',
+                    'class' => 'btn-outline-secondary new-product-button' . (empty($productId) ? ' disabled' : ''),
```

Comment:
```suggestion
                    'class' => 'btn-outline-secondary new-product new-product-button' . (empty($productId) ? ' disabled' : ''),
```

Reply:
Woups, be careful with this `new-product` class it is very important because if I remember correctly it is used as a selector to initialize the create product modal So If you change it this way I'm afraid the modal doesn't work anymore